internal/server/repository_worker.go

```diff
@@ -117,20 +118,22 @@ func lookupWorker(ctx context.Context, reader db.Reader, id string) (*Worker, er
 	case id == "":
 		return nil, errors.New(ctx, errors.InvalidParameter, op, "id is empty")
 	}
-	wAgg := &workerAggregate{}
-	wAgg.PublicId = id
-	err := reader.LookupById(ctx, wAgg)
+
+	lookupQuery := fmt.Sprintf("%s where w.public_id = @worker_id", listWorkersQuery)
```

Comment:
I think we should make a complete `lookupQuery` const in `query.go` instead of using Sprintf here, even if that means duplicating some of the sql.